Dockerfile.production

```
@@ -0,0 +1,37 @@
+# Production Dockerfile for ch11 Bankruptcy API
+FROM python:3.10-slim
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)